dipy/sims/voxel.py

```diff
@@ -15,6 +15,17 @@
 #
 diffusion_evals = np.array([1500e-6, 400e-6, 400e-6])
 
+# DT and KT for a voxel of well aligned fibers 
+# parameters are based on the simulations from:
+#
+#    Neto Henriques et al., "Exploring the 3D geometriy of the diffusion 
+#    kurtosis tensor - Impact on the development of robust tractography 
+#    procedures and novel biomarkers", NeuroImage, 2015; 111:85-99.
+#
+diffusion_tensor = np.array([1638e-6, 444e-6, 444e-6, 0, 0, 0])
```

Comment:
Would it be OK to use the same diffusion evals as the ones just above (on L16)? It's starting to get crowded in this namespace...